docs/components/tabs.md

```diff
@@ -220,6 +220,7 @@ The following example shows a row of of scrollable tabs.
 `Tabs`
```

Comment:
There needs to be a "Types" section with the types numbered in-line instead of a numbered list.

Reply:
Done